generators/server/templates/pom.xml.ejs

```diff
@@ -103,11 +103,9 @@
   <%_ if (!reactive) { _%>
         <liquibase-hibernate5.version><%= LIQUIBASE_VERSION %></liquibase-hibernate5.version>
   <%_ } _%>
-  <%_ if (devDatabaseTypeH2Any) { _%>
```

Comment:
There is a reference to h2.version here: https://github.com/jhipster/generator-jhipster/blob/8cb356615a28c21e128f1c6f3fab8c8faf65f8b7/generators/server/templates/pom.xml.ejs#L1779